src/examples/14-more-inline-elements.html

```diff
@@ -1,160 +0,0 @@
-<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
```

Comment:
I think a link to this will need to be removed from `src/examples/index.html`, right?